composer.json

```diff
@@ -15,7 +14,7 @@
         "symfony/translation": "~2.3",
         "pagerfanta/pagerfanta": "dev-master",
         "franmomu/silex-pagerfanta-provider": "dev-master",
-        "doctrine/dbal": "2.2.*",
+        "doctrine/dbal": "v2.5.1",
```

Comment:
Sorry about that, I just used the way it's shown in [Packagist](https://packagist.org/packages/doctrine/dbal#v2.5.1). Seems like they added the `v` from v2.4.0.

Bumping DBAL was required for the 
```
->setParameter(':name', $slugs, \Doctrine\DBAL\Connection::PARAM_STR_ARRAY);
```
 line to work correctly. 

I think the right way to require it would actually be: `"doctrine/dbal": "^v2.5.1"` to stick to [non breaking update](https://getcomposer.org/doc/articles/versions.md#caret).